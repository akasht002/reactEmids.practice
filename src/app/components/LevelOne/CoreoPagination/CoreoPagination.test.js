import React from 'react';
import { CoreoPagination } from '../index';
import renderer from 'react-test-renderer';

test('CoreoPagination Component Testing', () => {
    const component = renderer.create(
        <CoreoPagination
            activePage={1}
            itemsCountPerPage={10}
            totalItemsCount={1000}
            pageRangeDisplayed={20}
            itemClass='PaginationItem'
            itemClassFirst='PaginationIcon First'
            itemClassPrev='PaginationIcon Prev'
            itemClassNext='PaginationIcon Next'
            itemClassLast='PaginationIcon Last'>
        </CoreoPagination>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});

