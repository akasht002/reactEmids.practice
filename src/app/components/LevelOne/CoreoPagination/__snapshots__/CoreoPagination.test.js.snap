// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CoreoPagination Component Testing 1`] = `
<div
  className="col-md-12 p-0 CoreoPagination"
>
  <ul
    className="pagination"
  >
    <li
      className="PaginationItem PaginationIcon First disabled"
      onClick={[Function]}
    >
      <a
        className=""
        href="#"
      >
        «
      </a>
    </li>
    <li
      className="PaginationItem PaginationIcon Prev disabled"
      onClick={[Function]}
    >
      <a
        className=""
        href="#"
      >
        ⟨
      </a>
    </li>
    <li
      className="PaginationItem active"
      onClick={[Function]}
    >
      <a
        className="undefined"
        href="#"
      >
        1
      </a>
    </li>
    <li
      className="PaginationItem"
      onClick={[Function]}
    >
      <a
        className=""
        href="#"
      >
        2
      </a>
    </li>
    <li
      className="PaginationItem"
      onClick={[Function]}
    >
      <a
        className=""
        href="#"
      >
        3
      </a>
    </li>
    <li
      className="PaginationItem"
      onClick={[Function]}
    >
      <a
        className=""
        href="#"
      >
        4
      </a>
    </li>
    <li
      className="PaginationItem"
      onClick={[Function]}
    >
      <a
        className=""
        href="#"
      >
        5
      </a>
    </li>
    <li
      className="PaginationItem"
      onClick={[Function]}
    >
      <a
        className=""
        href="#"
      >
        6
      </a>
    </li>
    <li
      className="PaginationItem"
      onClick={[Function]}
    >
      <a
        className=""
        href="#"
      >
        7
      </a>
    </li>
    <li
      className="PaginationItem"
      onClick={[Function]}
    >
      <a
        className=""
        href="#"
      >
        8
      </a>
    </li>
    <li
      className="PaginationItem"
      onClick={[Function]}
    >
      <a
        className=""
        href="#"
      >
        9
      </a>
    </li>
    <li
      className="PaginationItem"
      onClick={[Function]}
    >
      <a
        className=""
        href="#"
      >
        10
      </a>
    </li>
    <li
      className="PaginationItem"
      onClick={[Function]}
    >
      <a
        className=""
        href="#"
      >
        11
      </a>
    </li>
    <li
      className="PaginationItem"
      onClick={[Function]}
    >
      <a
        className=""
        href="#"
      >
        12
      </a>
    </li>
    <li
      className="PaginationItem"
      onClick={[Function]}
    >
      <a
        className=""
        href="#"
      >
        13
      </a>
    </li>
    <li
      className="PaginationItem"
      onClick={[Function]}
    >
      <a
        className=""
        href="#"
      >
        14
      </a>
    </li>
    <li
      className="PaginationItem"
      onClick={[Function]}
    >
      <a
        className=""
        href="#"
      >
        15
      </a>
    </li>
    <li
      className="PaginationItem"
      onClick={[Function]}
    >
      <a
        className=""
        href="#"
      >
        16
      </a>
    </li>
    <li
      className="PaginationItem"
      onClick={[Function]}
    >
      <a
        className=""
        href="#"
      >
        17
      </a>
    </li>
    <li
      className="PaginationItem"
      onClick={[Function]}
    >
      <a
        className=""
        href="#"
      >
        18
      </a>
    </li>
    <li
      className="PaginationItem"
      onClick={[Function]}
    >
      <a
        className=""
        href="#"
      >
        19
      </a>
    </li>
    <li
      className="PaginationItem"
      onClick={[Function]}
    >
      <a
        className=""
        href="#"
      >
        20
      </a>
    </li>
    <li
      className="PaginationItem PaginationIcon Next"
      onClick={[Function]}
    >
      <a
        className=""
        href="#"
      >
        ⟩
      </a>
    </li>
    <li
      className="PaginationItem PaginationIcon Last"
      onClick={[Function]}
    >
      <a
        className=""
        href="#"
      >
        »
      </a>
    </li>
  </ul>
</div>
`;
